### PR TITLE
[GraFx Studio] Media Connector for Acquia Dam

### PR DESCRIPTION
**Pull request by BramVerniest**

### CODE REVIEW

File: docs/GraFx-Studio/concepts/connectors-media/index.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### Built by CHILI publish
+
+To facilitate connections to external services, CHILI publish develops Media Connectors for specific applications. For instance, the Media Connector for Acquia DAM is built and supported by CHILI publish. Support for these connectors encompasses their internal workings, including API calls made to external systems. However, issues related to availability, configuration, or API problems on the media provider's side are not covered under support. Our Customer Support team assists in identifying the origin of such issues.
+
+**Example**: The Media Connector for Acquia DAM
+
+### Built by Third Party, Approved by CHILI publish
+
+Our open connector framework encourages developers to [build connectors](/GraFx-Developers/connectors/build-media-connector/) for their unique use cases. If a media provider develops a connector, such as for a generative AI application, it can be offered in our marketplace. Support for these connectors is provided by the developer.
```
Comment (jeroencosyn) [inline docs/GraFx-Studio/concepts/connectors-media/index.md:23]: Marketplace should be Connector Hub? @Pieterjan-Blondeel, can you confirm?

File: docs/GraFx-Studio/concepts/connectors-media/connector2.png
Comment (jeroencosyn) [inline docs/GraFx-Studio/concepts/connectors-media/connector2.png:1]: Is it a good idea to put Acquia in this picture? You might think we can only connect with the Acquia DAM.

**Review by jeroencosyn — APPROVED** [review]
Looks good to me, but I would also wait on @Pieterjan-Blondeel's approval before merging it.

File: docs/GraFx-Studio/concepts/connectors-media/connector2.png
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/concepts/connectors-media/connector2.png:1]: @BramVerniest I would show acquia logo in this visual, don't show any logo at all (the corporate website and documentation will always be updated once there is a new connector)

File: docs/GraFx-Studio/concepts/connectors-media/index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+![ui](connector2.png)
+
+## Support
+
+CHILI publish categorizes CHILI GraFx Connectors into four categories.
+
+### Built-in
+
+GraFx Studio connects to GraFx Media through the GraFx Media connector, utilizing the same framework as other media connectors. CHILI publish assumes full responsibility for the development and support of the GraFx Media connector and the associated media service, as all elements were built in-house.
```
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/concepts/connectors-media/index.md:13]: @BramVerniest maybe link to https://docs.chiligrafx.com/GraFx-Media/?

File: docs/GraFx-Studio/concepts/connectors-media/index.md (hunk 1 above)
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/concepts/connectors-media/index.md:23]: , such as for a generative AI application, --> I would leave this out? Doesn't add much to the explaination?

Comment (Pieterjan-Blondeel) [reply docs/GraFx-Studio/concepts/connectors-media/index.md:23]: Correct

File: docs/GraFx-Studio/guides/connect-acquia/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+| --- | --- |
+|  | Built-in |
+| :fontawesome-regular-square-check: | Built by CHILI publish |
+|  | Third party |
+
+[See Connector types](/GraFx-Studio/concepts/connectors-media/#support)
+
+## Activation
+
+Once a connector is available in the GraFx Connector Hub, it can be activated on your environment.
```
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/guides/connect-acquia/index.md:13]: GraFx Connector Hub --> remove 'GraFx'

File: docs/GraFx-Studio/concepts/connectors-media/index.md (hunk 1 above)
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/concepts/connectors-media/index.md:23]: I would add that this a connector that only will be available within a single environment

File: docs/GraFx-Studio/concepts/connectors/index.md
Comment (Pieterjan-Blondeel) [inline docs/GraFx-Studio/concepts/connectors/index.md:1]: should we rename 'DAM' to media provider in this picture? What do you think? @BramVerniest

File: docs/GraFx-Studio/concepts/connectors-media/index.md (hunk 1 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/concepts/connectors-media/index.md:84]: corrected

File: docs/GraFx-Studio/concepts/connectors-media/connector2.png
Comment (BramVerniest) [reply docs/GraFx-Studio/concepts/connectors-media/connector2.png:1]: removed it

Comment (BramVerniest) [reply docs/GraFx-Studio/concepts/connectors-media/connector2.png:1]: I guess you mean would not show? Anyway, it's removed, to only show the concept

File: docs/GraFx-Studio/concepts/connectors-media/index.md (hunk 2 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/concepts/connectors-media/index.md:74]: done

File: docs/GraFx-Studio/concepts/connectors-media/index.md (hunk 1 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/concepts/connectors-media/index.md:84]: removed

File: docs/GraFx-Studio/guides/connect-acquia/index.md (hunk 3 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/guides/connect-acquia/index.md:13]: removed

File: docs/GraFx-Studio/concepts/connectors-media/index.md (hunk 1 above)
Comment (BramVerniest) [reply docs/GraFx-Studio/concepts/connectors-media/index.md:84]: isn't that the case for any connector?

Comment (BramVerniest) [reply docs/GraFx-Studio/concepts/connectors-media/index.md]: Let's revisit in a future update, so this can go live

**Pull request merged**